### PR TITLE
feat: Overview Page Display the programs portlet - MEED-674 - Meeds-io/MIPs#10  

### PR DESCRIPTION
**Pull request by MayTekayaa**
This change will add the programs portlet positionned in the sixth position. It contains the title of the portlet and three listed programs.

### CODE REVIEW

File: portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue [hunk 1]
```diff
@@ -55,6 +99,18 @@ export default {
     clickOnEmptyActionLink() {
       window.location.href = `${eXo.env.portal.context}/${eXo.env.portal.portalName}/contributions/challenges`;
     },
+    retrievePrograms() {
+      return this.$programsServices
+        .retrievePrograms(0, 3, this.type, this.status)
+        .then((data) => {
+          this.programs = data.domains;
+          this.programsDisplayed = data.domains.length > 0;
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue]:
```suggestion
          this.programsDisplayed = data.domainsSize > 0;
```

File: portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue [hunk 2]
```diff
@@ -55,6 +99,18 @@ export default {
     clickOnEmptyActionLink() {
       window.location.href = `${eXo.env.portal.context}/${eXo.env.portal.portalName}/contributions/challenges`;
     },
+    retrievePrograms() {
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue:102]: Can you add preload.resource.rest in the porlet config please!

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue (hunk 1 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue]: done!

File: portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue (hunk 2 above)
Comment (MayTekayaa) [reply portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue:102]: done!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**